eng/common/scripts/ChangeLog-Operations.ps1

```diff
@@ -261,11 +275,12 @@ function Set-ChangeLogContent {
     [Parameter(Mandatory = $true)]
     [String]$ChangeLogLocation,
     [Parameter(Mandatory = $true)]
-    $ChangeLogEntries
+    $ChangeLogEntries,
+    [String]$InitialAtxHeader="#"
```

Comment:
Do we need to pass it in here? As we should have it on ChangeLogEntries already, if it isn't there we can default to #